### ISSUE TITLE
Panel plugin with id corpglory-multibar-graph-panel could not be found

### ISSUE THREAD

**Issue by mxk1235** [CorpGlory/grafana-multibar-graph-panel#34]
I am seeing `Panel plugin with id corpglory-multibar-graph-panel could not be found` when trying to add a the multibar panel.  

Running in a new grafana/grafana container, installed via GF_INSTALL_PLUGINS environment variable

```grafana_1     | installing multibar-graph-panel @
grafana_1     | from url: https://github.com/CorpGlory/grafana-multibar-graph-panel/archive/0.2.4.zip
grafana_1     | into: /var/lib/grafana/plugins
grafana_1     |
grafana_1     | ✔ Installed multibar-graph-panel successfully
grafana_1     |
grafana_1     | Restart grafana after installing plugins . <service grafana-server restart>
grafana_1     |
grafana_1     | t=2019-03-09T00:36:37+0000 lvl=info msg="Starting Grafana" logger=server version=6.0.1 commit=0c44a04 branch=HEAD compiled=2019-03-06T14:21:49+0000```

**Comment by jonyrock** [comment]
Hey @mxk1235 we don't support this plugin. Please let us know ping@corpglory.com if you need this issue fixed.

**Comment by Fank** [comment]
@mxk1235 do you mean this?
![image](https://user-images.githubusercontent.com/1900106/55634276-703d9e80-57be-11e9-9482-689768d4c9ef.png)

**Comment by jonyrock** [comment]
@mxk1235 I think the problem is that you are using this on Grafana 6 which we haven't tested

**Comment by mxk1235** [comment]
@Fank yes, that's what i'm seeing.

@jonyrock i'm seeing it with Grafana 5 and 6 unfortunately

**Comment by jonyrock** [comment]
@Fank if you ready to sponsor this feature support, please ping@corpglory.com

**Comment by rozetko** [comment]
Hey, @mxk1235 
It is fixed in the latest `0.2.5` release
You can use this URL in `GF_INSTALL_PLUGINS` to install the plugin: https://github.com/CorpGlory/grafana-multibar-graph-panel/archive/0.2.5.zip
